swarm_examples/README.md

```diff
@@ -3,18 +3,25 @@ Introduction
 
 This is an example demonstrating swarming for prediction and anomaly detection
 models. The example also demonstrates using multiple fields. We assume you have
-read the following wiki page:
+read the following page:
 
-https://github.com/numenta/nupic/wiki/Running-Swarms
+http://nupic.docs.numenta.org/0.7.0.dev0/guides/swarming/running.html
 
 DISCLAIMER: we are using sine waves below to demonstrate some of the points
 because this seems to be one of the most popular first tasks that people try.
-Sine waves are actually not ideal for the CLA. It is a single continuously 
+Sine waves are actually not ideal for the CLA. It is a single continuously
 repeating pattern with no clear beginning and end. Please see an email
 from Jeff Hawkin on this topic:
 
 http://lists.numenta.org/pipermail/nupic_lists.numenta.org/2013-June/000327.html
 
+Assumptions
+===========
+
+You have [NuPIC](http://github.com/numenta/nupic) 0.6.0 installed and the `NUPIC`
+environment variable set to the repository path.
+
+    pip install nupic==0.6.0
```

Comment:
Maybe leave out NuPIC installation instructions and just put in a link to NuPIC?